cves/CVE-2004-0751.yml

```
@@ -94,11 +102,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
-  contest:
+  answer: |
+    Found using a RewriteRule to do reverse proxying to an SSL server running IIS.
+    To reproduce, certain directives are required in httpd.conf. 
+  date: 2004-07-15
```

If you search for the CVE here: [https://httpd.apache.org/security/vulnerabilities_20.html](url)
It looks like the date was 7th July 2004
```suggestion
  date: 2004-07-07
```

---

I was going off of this link: [https://bz.apache.org/bugzilla/show_bug.cgi?id=30134](url)
The reported date is 15th July 2004, but I will go with the link you provided since it is an earlier date.